EduNLP/Formula/Formula.py

```diff
@@ -15,6 +15,30 @@
 
 class Formula(object):
     """
+    The part transform a formula to the parsed abstracted syntax tree.
+
+    Attributes
```

Comment:
The order of comments can adjust to : discription, Parameters , Examples, Attributes. When showed in doc, the attributes is always close to the functions in Class.

Reply:
FoIlowing your advise above all, I have made some updates. Please check it so that I can prettify it better. Thanks.